pants-plugins/BUILD

```
@@ -2,7 +2,7 @@ __defaults__(
     all=dict(
         resolve="pants-plugins",
         skip_pylint=True,
-        interpreter_constraints=["CPython==3.9.*"],
+        interpreter_constraints=["CPython<=3.11,>=3.8"],
```

This code only runs in pants, so it should match pants constraints. 

```suggestion
        interpreter_constraints=["CPython==3.9.*"],
```

BTW Pants will jump to python 3.11 in a future version. Once we update up that version, we can adjust this constraint.

---

I had a hard time with pants, so take all those settings with that in mind.  `:insert dog typing on keyboard meme here:`